snapcraft/parts/update_metadata.py

```diff
@@ -50,6 +50,29 @@ def update_project_metadata(
     """
     _update_project_variables(project, project_vars)
 
+    update_from_extracted_metadata(
```

Comment:
Don't forget to update the copyright line year in line 3.

Reply:
nice catch thanks!